moped-database/migrations/1651001224000_alter_project_list_view_funding/up.sql

```diff
@@ -0,0 +1,96 @@
+drop view project_list_view;
+
+CREATE VIEW public.project_list_view AS
+with project_person_list_lookup AS (
+  SELECT
+    mpp.project_id,
+    string_agg(DISTINCT concat(mu.first_name, ' ', mu.last_name, ':', mpr.project_role_name), ',') AS project_team_members
+   FROM 
+    public.moped_proj_personnel mpp
+        JOIN public.moped_users mu ON mpp.user_id = mu.user_id
+        JOIN public.moped_project_roles mpr ON mpp.role_id = mpr.project_role_id
+      WHERE (mpp.status_id = 1)
+      GROUP BY mpp.project_id
+),
+ funding_sources_lookup AS (
+  SELECT
+    mpf.project_id,
+    string_agg(mfs.funding_source_name, ', ') AS funding_source_name
```

Comment:
Thank you for testing that! I like the idea of using DISTINCT. I don't know if users will be including the same funding source on multiple lines, but just in case I think we should write code to anticipate that.